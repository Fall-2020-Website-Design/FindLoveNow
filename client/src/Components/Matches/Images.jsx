import React, { Component } from 'react';
import Row from 'react-bootstrap/Row';
import Col from 'react-bootstrap/Col';
import Image from 'react-bootstrap/Image';
import './Matches.css';

export class Images extends Component {

    render() {
        return (
            <div>
                <Row className="justify-content-md-center">
                    <Col>
                        <Image src={this.props.image} className="card-img" />
                    </Col>
                </Row>
            </div>
        )
    }
}

export default Images
