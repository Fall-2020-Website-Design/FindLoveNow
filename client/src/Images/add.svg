<svg width="127" height="127" viewBox="0 0 127 127" fill="none" xmlns="http://www.w3.org/2000/svg">
<g filter="url(#filter0_d)">
<circle cx="63.5" cy="59.5" r="38.5" fill="#DFDFDF"/>
</g>
<line x1="64.5" y1="32" x2="64.5" y2="82" stroke="#EE7E7E" stroke-width="3"/>
<line x1="88" y1="58.5" x2="38" y2="58.5" stroke="#EE7E7E" stroke-width="3"/>
<defs>
<filter id="filter0_d" x="0" y="0" width="127" height="127" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB">
<feFlood flood-opacity="0" result="BackgroundImageFix"/>
<feColorMatrix in="SourceAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"/>
<feOffset dy="4"/>
<feGaussianBlur stdDeviation="12.5"/>
<feColorMatrix type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.15 0"/>
<feBlend mode="normal" in2="BackgroundImageFix" result="effect1_dropShadow"/>
<feBlend mode="normal" in="SourceGraphic" in2="effect1_dropShadow" result="shape"/>
</filter>
</defs>
</svg>
