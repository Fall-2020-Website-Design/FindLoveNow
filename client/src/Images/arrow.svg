<svg width="48" height="48" viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
<g filter="url(#filter0_d)">
<rect x="4" width="40" height="40" fill="url(#pattern0)" fill-opacity="0.5"/>
<rect x="4.5" y="0.5" width="39" height="39" stroke="black"/>
</g>
<defs>
<filter id="filter0_d" x="0" y="0" width="48" height="48" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB">
<feFlood flood-opacity="0" result="BackgroundImageFix"/>
<feColorMatrix in="SourceAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"/>
<feOffset dy="4"/>
<feGaussianBlur stdDeviation="2"/>
<feColorMatrix type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.25 0"/>
<feBlend mode="normal" in2="BackgroundImageFix" result="effect1_dropShadow"/>
<feBlend mode="normal" in="SourceGraphic" in2="effect1_dropShadow" result="shape"/>
</filter>
<pattern id="pattern0" patternContentUnits="objectBoundingBox" width="1" height="1">
<use xlink:href="#image0" transform="scale(0.00195312)"/>
</pattern>
<image id="image0" width="512" height="512" xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAgAAAAIACAYAAAD0eNT6AAAAAXNSR0IArs4c6QAAHdhJREFUeAHt3d2PrWdVAPCetrRQOAWjQkuE0oKUBMQEwoUk0la94KMhXpiAChIDCQnemIBeqDFcCMaEf0CCUeA0oXABEfkI4SvBJiTcSLjACvJlogIFVChF8JzjeqJNTifnzDuz9157v+tZvzdZcGbm3c/7rN9607Vm75k9V13lIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIEDgFAJnTnGuUwkQILAvgevjQk+KOBsx/jv1UMS3Ir4f4SBAYAcCBoAdIFqCAIGtBW6NFe6OuCPi+RG3RFzuv0/fjs9/PuK+iA9HfC7iYoSDAAECBAgQKCLwqNjnqyJGMx9NfJP4ajzujyN+OsJBgAABAgQIrFzglbG/r0Rs0vQv95jx0sCbI26IcBAgQIAAAQIrE/i52M/HIi7XxHfxuTFU3LGynG2HAAECBAi0Fvi1yP6BiF00+uPWOB/XGC8LOAgQIECAAIEDC7w6rv+TiOMa966/9tdxvWsOnLfLEyBAgACBtgKj+V+I2HWDP8l674nrGgLa3noSJ0CAAIFDCYyn/ff9nf/RwcAQcKjquy4BAgQItBR4SmT9nYijDfkQHxsCWt6CkiZAgACBQwh8PC56iGZ/pWsaAg5xF7gmAQIECLQS+K3I9kqN+JCfNwS0ug0lS4AAAQL7FBjv8Pe1iEM2+uOubQjY593gWgQIECDQRuB3ItPjGvAavmYIaHM7SpQAAQIE9iXw2bjQGpr80h4MAfu6I1yHAAECBKYXeHpkuNR41/R1Q8D0t6QECRAgQGAfAr8fF1lTgz/JXgwB+7gzXIMAAQIEphZ4f2R3kqa7tnMMAVPflpIjQIAAgWyBb8QF1tbcT7ofQ0D23WF9AgQIEJhS4DGR1Umb7VrPMwRMeWtKigABAgQyBW6Nxdfa2E+zL0NA5l1ibQIECBCYTuC5kdFpGu2azzUETHd7SuiowNVHP+FjAgQIbChwZsPHrfFhr4hN3RPhTwmvsTr2RIAAAQKrEnhm7GbN39VvsjfPBKzqFrMZAgQIEFijwBNiU5s02bU/xhCwxrvNnggQIEBgVQLfid2svaFvsj9DwKpuM5shQIAAgbUJfDI2tEmDrfAYQ8Da7jb72UrADwFuxefBBAgcEbjvyMczfegHA2eqplwIECBAYKcCL4zVKnw3v80ePROw01vGYgQIECAwg8D4VcCvR2zTYCs81hAww93aPAe/49r8BpA+gQSBs7HmXQnrrmnJ58Rmbo94+I8frWlv9kKAAAECBA4i8DNx1R9EVPhOfts9eibgILeYi+5CwDMAu1C0BgEClwr8MD54dMQdl35y0n97JmDSwkqLAAECBDYTuCEe9tWIbb/DrvJ4zwRsdp94FAECBAhMKDB+DuB8RJUmvu0+DQET3sQzp+QlgJmrKzcChxX4Wlz+QsSvHHYbe7u6lwP2Ru1CBAgQIFBB4G9ik9t+d13p8Z4JqHBX2iMBAgQIpAuMZxpHU6zUxLfdqyEg/bZyAQIECBCoIGAIqFAleyRAgAABAgkChoAEVEsSIECAAIEKAoaAClWyRwIECBAgkCBgCEhAtSQBAgQIEKggYAioUCV7JECAAAECCQKGgARUSxIgQIAAgQoChoAKVbJHAgQIECCQIGAISEC1JAECBAgQqCBgCKhQJXskQIAAAQIJAoaABFRLEiBAgACBCgKGgApVskcCBAgQIJAgYAhIQLUkAQIECBCoIGAIqFAleyRAgAABAgkChoAEVEsSIECAAIEKAoaAClWyRwIECBAgkCBgCEhAtSQBAgQIEKggYAioUCV7JECAAAECCQKGgARUSxIgQIAAgQoChoAKVbJHAgQIECCQIGAISEC1JAECBAgQqCBgCKhQJXskQIAAAQIJAoaABFRLEiBAgACBCgKGgApVskcCBAgQIJAgYAhIQLUkAQIECBCoIGAIqFAleyRAgAABAgkChoAEVEsSIECAAIEKAoaAClWyRwIECBAgkCBgCEhAtSQBAgQIEKggYAioUCV7JECAAAECCQKGgARUSxIgQIAAgQoChoAKVbJHAgQIECCQIGAISEC1JAECBAgQqCBgCKhQJXskQIAAAQIJAoaABFRLEiBAgACBCgKGgApVskcCBAgQIJAgYAhIQLUkAQIECBCoIGAIqFAleyRAgAABAgkChoAEVEsSIECAAIEKAoaAClWyRwIECBAgkCBgCEhAtSQBAgQIEKggYAioUCV7JECAAAECCQKGgARUSxIgQIAAgQoChoAKVbJHAgQIECCQIGAISEC1JAECBAgQqCBgCKhQJXskQIAAAQIJAoaABFRLEiBAgACBCgKGgApVskcCBAgQIJAgYAhIQLUkAQIECBCoIGAIqFAleyRAgAABAgkChoAEVEsSIECAAIEKAoaAClWyRwIECBAgkCBgCEhAtSQBAgQIEKggYAioUCV7JECAAAECCQKGgARUSxIgQIAAgQoChoAKVbJHAgQIECCQIGAISEC1JAECBAgQqCBgCKhQJXskQIAAAQIJAoaABFRLEiBAgACBCgKGgApVskcCBAgQIJAgYAhIQLUkAQIECBCoIGAIqFAleyRAgAABAgkChoAEVEsSIECAAIEKAoaAClWyRwIECBAgkCBgCEhAtSQBAgQIEKggYAioUCV7JECAAAECCQKGgARUSxIgQIAAgQoChoAKVbJHAgQIECCQIGAISEC1JAECBAgQqCBgCKhQJXskQIAAAQIJAoaABFRLEiBAgACBCgKGgApVskcCBAgQIJAgYAhIQLUkAQIECBCoIGAIqFAleyRAgAABAgkChoAEVEsSIECAAIEKAoaAClWyRwIECBAgkCBgCEhAtSQBAgQIEKggYAioUCV7JECAAAECCQKGgARUSxIgQIAAgQoChoAKVbJHAgQIECCQIGAISEC1JAECBAgQqCBgCKhQJXskQIAAAQIJAoaABFRLEiBAgACBCgKGgApVskcCBAgQIJAgYAhIQLUkAQIECBCoIGAIqFAleyRAgAABAgkChoAEVEsSIECAAIEKAoaAClWyRwIECBAgkCBgCEhAtSQBAgQIEKggYAioUCV7JECAAAECCQKGgARUSxIgQIAAgQoChoAKVbJHAgQIECCQIGAISEC1JAECBAgQqCBgCKhQJXskQIAAAQIJAoaABFRLEiBAgACBCgKGgApVskcCBAgQIJAgYAhIQLUkAQIECBCoIGAIqFAleyRAgAABAgkChoAEVEsSIECAAIEKAoaAClWyRwIECBAgkCBgCEhAtSQBAgQIEKggYAioUCV7JECAAAECCQKGgARUSxIgQIAAgQoChoAKVbJHAgQIECCQIGAISEC1JAECBAgQqCBgCKhQJXskQIAAAQIJAoaABFRLEiBAgACBCgKGgApVskcCBAgQIJAgYAhIQLUkAQIECBCoIGAIqFAleyRAgAABAgkChoAEVEsSIECAAIEKAoaAClWyRwIECBAgkCBgCEhAtSQBAgQIEKggYAioUKVT7vHMKc+f4fQbIomfj3hixNmI6yIcBAgQIHC8wBgC3hrx1ONPm+qr90Y2vx1xfqqs/j+ZDgPAoyPXuyNeHHFnxG0RHfKONB0ECBAgsKXAtEPAzI3wlij6myJeHfH4LW8ADydAgACBvgJTDgEzDgBPiHv0zyJeH3Ft3/tV5gQIECCwQ4HphoDxms5Mx8simY9H3BVx9UyJyYUAAQIEDirwnLj67RHvj7h40J3s6OKzNMnxTMafR3wwYvxwn4MAAQIECOxa4BWx4D0RU3zzPEMS42n+d0a8IWLGlzQiLQcBAgQIrERgmmcCqg8Ao+GP5v+qldwYtkGAAAEC8wuMIeDWiA9UTrX6ADCe9h/f+TsIECBAgMA+BX4xLja+Cf30Pi+6y2uNzVc9xg/8jdf8K+dQ1d6+CRAgQOD/fhhwvMfMxypiVG2e41f97o/wA38V7zp7JkCAwDwC/xKpjN8OeKhaSlVfAnhbQN9VDdt+CRAgQGA6gfFGc/8VcV+1zCo+AzDe4e/LEd7kp9rdZr8ECBCYU+A/Iq3xN2YeqJRexfcBGG/vq/lXusvslQABAnMLjJel31gtxWrPAIw/7PPvEd7bv9qdZr8ECBCYW+Bbkd7NEReqpFntGYC7A1bzr3J32ScBAgT6CIwfSn9BpXSrDQDj1y0cBAgQIEBgjQIvX+OmrrSnagPAnVdKxOcJECBAgMCBBUoNAJV+BuCGKOwPIirt+cD3ossTIECAwB4Fxuv/j4340R6vufGlKj0DMH7FQvPfuNQeSIAAAQLJAqOnPi35GjtbvtIA4F3/dlZ2CxEgQIBAksBNSevufNlKA8DZnWdvQQIECBAgsFuBx+12ubzVKg0AVd+2OK96ViZAgACBtQmUeaO6SgPAg2ursv0QIECAAIEjAt8/8vFqP6w0AIx3AHQQIECAAIE1C5TpVZV+qv7GqPh/rrnq9kaAAAECrQX8GmBS+cefW/xK0tqWJUCAAAEC2wp8IRYo8R4AI9FKLwGM/X5m/I+DAAECBAisUOBvV7inK26p2gDwoStm4gsECBAgQOCwAn932Muf7uqVfgZgZDbeDvjfIsbPAzgIECBAgMBaBL4ZG3lyhD8HnFSRH8a6705a27IECBAgQGBTgb+KB5Zp/iPJas8AjD3fFnF/RJk3WxibdhAgQIDAtALfjcyeEfG9ShlW+xmAYTt+E+DtlZDtlQABAgSmFnhrZFeq+Y9qVHwGYOz7pyK+GPGk8YGDAAECBAgcSOAbcd1nRTx0oOtvfNmKzwCMZMek9dqIi+MDBwECBAgQOIDAeM3/dyPKNf9hVfkP7Hwp9n9dxC+PRBwECBAgQGDPAn8S13vXnq+5s8tVHgAGwqcixg9ePHd84CBAgAABAnsSeEdc5w/3dK2Uy1QfAAbKeOel2yIMAUPDQYAAAQLZAvfEBV4XUfpl6BkGgPEazAciRi7j5YCqP9gYW3cQIECAwMoFRvN/TcT5le9zcXuzNcuXRMbjzRhuXszcCQQIECBA4HQC0zT/kfYMzwBcWr4vxwdjAHhMxPMiZssvUnIQIECAwAEEpmr+w2/GBjn+FONHI8ZPZo5nOG6PGAOBgwABAgQIbCIwXfMfCLO9BHC5wl4fn3xxxEsjXhTxzIiq738QW3cQIECAwB4Fpmz+w6/DAHD0PhnPBozfGrgp4myEvykQCA4CBAgsCIxnjP8i4paF82b68rTNf6YiyYUAAQIE8gRG8783YvzaW5c4F7nO+DJ5pOUgQIAAAQLLApr/spEzCBAgQIDAVAKa/1TllAwBAgQIEFgW0PyXjZxBgAABAgSmEtD8pyqnZAgQIECAwLKA5r9s5AwCBAgQIDCVgOY/VTklQ4AAAQIElgU0/2UjZxAgQIAAgakENP+pyikZAgQIECCwLKD5Lxs5gwABAgQITCWg+U9VTskQIECAAIFlAc1/2cgZBAgQIEBgKgHNf6pySoYAAQIECCwLaP7LRs4gQIAAAQJTCWj+U5VTMgQIECBAYFlA8182cgYBAgQIEJhKQPOfqpySIUCAAAECywKa/7KRMwgQIECAwFQCmv9U5ZQMAQIECBBYFtD8l42cQYAAAQIEphLQ/Kcqp2QIECBAgMCygOa/bOQMAgQIECAwlYDmP1U5JUOAAAECBJYFNP9lI2cQIECAAIGpBDT/qcopGQIECBAgsCyg+S8bOYMAAQIECEwloPlPVU7JECBAgACBZYFr45R7Iy42inOR6xh6HAQIECBAoKWA5t+y7JImQIAAgc4Cmn/n6sudAAECBFoKaP4tyy5pAgQIEOgsoPl3rr7cCRAgQKClgObfsuySJkCAAIHOApp/5+rLnQABAgRaCmj+LcsuaQIECBDoLKD5d66+3AkQIECgpYDm37LskiZAgACBzgKaf+fqy50AAQIEWgpo/i3LLmkCBAgQ6Cyg+XeuvtwJECBAoKWA5t+y7JImQIAAgc4Cmn/n6sudAAECBFoKaP4tyy5pAgQIEOgsoPl3rr7cCRAgQKClgObfsuySJkCAAIHOApp/5+rLnQABAgRaCmj+LcsuaQIECBDoLKD5d66+3AkQIECgpYDm37LskiZAgACBzgKaf+fqy50AAQIEWgpo/i3LLmkCBAgQ6Cyg+XeuvtwJECBAoKWA5t+y7JImQIAAgc4Cmn/n6sudAAECBFoKaP4tyy5pAgQIEOgsoPl3rr7cCRAgQKClwGj+74242CjORa7XRDgIECBAgEBLAc2/ZdklTYAAAQKdBTT/ztWXOwECBAi0FND8W5Zd0gQIECDQWUDz71x9uRMgQIBASwHNv2XZJU2AAAECnQU0/87VlzsBAgQItBTQ/FuWXdIECBAg0FlA8+9cfbkTIECAQEsBzb9l2SVNgAABAp0FNP/O1Zc7AQIECLQU0Pxbll3SBAgQINBZQPPvXH25EyBAgEBLAc2/ZdklTYAAAQKdBTT/ztWXOwECBAi0FND8W5Zd0gQIECDQWUDz71x9uRMgQIBASwHNv2XZJU2AAAECnQU0/87VlzsBAgQItBTQ/FuWXdIECBAg0FlA8+9cfbkTIECAQEsBzb9l2SVNgAABAp0FNP/O1Zc7AQIECLQU0Pxbll3SBAgQINBZQPPvXH25EyBAgEBLAc2/ZdklTYAAAQKdBTT/ztWXOwECBAi0FND8W5Zd0gQIECDQWUDz71x9uRMgQIBAS4HR/N8XcbFRnItcr4lwECBAgACBlgKaf8uyS5oAAQIEOgto/p2rL3cCBAgQaCmg+bcsu6QJECBAoLOA5t+5+nInQIAAgZYCmn/LskuaAAECBDoLaP6dqy93AgQIEGgpoPm3LLukCRAgQKCzgObfufpyJ0CAAIGWApp/y7JLmgABAgQ6C2j+nasvdwIECBBoKaD5tyy7pAkQIECgs4Dm37n6cidAgACBlgKaf8uyS5oAAQIEOgto/p2rL3cCBAgQaCmg+bcsu6QJECBAoLOA5t+5+nInQIAAgZYCmn/LskuaAAECBDoLaP6dqy93AgQIEGgpoPm3LLukCRAgQKCzgObfufpyJ0CAAIGWApp/y7JLmgABAgQ6C2j+nasvdwIECBBoKaD5tyy7pAkQIECgs4Dm37n6cidAgACBlgKaf8uyS5oAAQIEOgto/p2rL3cCBAgQaCmg+bcsu6QJECBAoLOA5t+5+nInQIAAgZYCZyLrcxEXG8XI95oIBwECBAgQaCvw5shc829bfokTIECAQEeBX42kL0R0GQB859/xLpczAQIECDxC4LHx0dcjNP9HsPiAAAECBAjMLfCWSE/zn7vGsiNAgAABAo8Q+Nn46MGIDgOAp/0fUXofECBAgEBngT+N5DX/zneA3AkQIECgncD4tb8Or/37zr/drS1hAgQIEDhO4Jfii7N/96/5H3cH+FoZgavL7NRGCRCoIPCSCpvcYo/3xGNfE3F+izU8lAABAgQITCfwicho1mcAfOc/3e0qIQIECBDYlcADsdCMA4Dmv6s7xDoECBAgMJ3AjZGR5j9dWSVEgAABAgSOF3h6fHm2AcB3/sfX3FcJECBAgMBVvxAGMw0Amr+bmgABAgQInEDg2XHOLAOA5n+CgjuFAAECBAgMgadEzDAAaP7uZwIECBAgcAqB6+Lc8fvxlYcAzf8UBXcqAQIECBB4WOCf4x9VBwDN/+Eq+n8CBAgQIHBKgffG+RUHAM3/lIV2OgECBAgQuFTg9+KDagOA5n9pBf2bAAECBAhsIHBLPKbSAKD5b1BkDyFAgAABApcT+Ex8ssIQoPlfrno+R4AAAQIENhT4zXjc2gcAzX/D4noYAQIECBC4ksC18YUvRax1CND8r1Q5nydAgAABAlsK/EY8fo0DgOa/ZWE9nAABAgQILAl8KE5Y0xCg+S9VzNcJECBAgMAOBG6ONb4ZsYYhQPPfQUEtQYAAAQIETipwZ5z444hDDgGa/0mr5TwCBAgQILBDgfFbAYf6GwGa/w4LaSkCBAgQIHBagVfGA/47Yp/PBPxlXO/q027U+QQIECBAgMBuBe6I5fbxMwH/E9d50263bjUCBAgQIEBgG4Gb4sEfjMh6JuCfYu0XbrNBjyVAgAABAgTyBH49lv7HiF0NAt+Ltf4o4voIBwECBAgQILBigfH6/HjDoE9GXIjYZBi4Px73BxGPj3AQIHAKgTOnONepBAgQyBJ4ciz8sog7I54X8YyIayMuPcaA8K8Rn4/4+4iPRPxDhIMAgQ0EDAAboHkIAQLpAuPZgSdG3Bgx/v1gxLcjfhThIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAEC+xb4Xx/LNfKUTduFAAAAAElFTkSuQmCC"/>
</defs>
</svg>
