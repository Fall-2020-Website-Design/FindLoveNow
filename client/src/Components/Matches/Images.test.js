import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { Images } from './Images';


Enzyme.configure({ adapter: new Adapter() });

describe('Images', () => {
	let wrapper;

	beforeEach(() => {
		wrapper = shallow(<Images />);
    });
    
    test('renders', () => {
		const component = renderer.create(
			<MemoryRouter>
				<Images  />
			</MemoryRouter>
		).toJSON();
		
		expect(component).toMatchSnapshot();
    });
    

});