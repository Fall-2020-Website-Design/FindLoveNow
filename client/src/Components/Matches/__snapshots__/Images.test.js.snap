// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Images renders 1`] = `
<div>
  <div
    className="justify-content-md-center row"
  >
    <div
      className="col"
    >
      <img
        className="card-img"
        src={null}
        style={
          Object {
            "height": "37rem",
          }
        }
      />
    </div>
  </div>
</div>
`;
